// juniper.afi_cap_action is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap-entry.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_cap_action;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";

message AfiCapAction {
  ywrapper.BoolValue copy_to_cpu = 503826992;
  ywrapper.BoolValue copy_to_cpu_cancel = 530999003;
  ywrapper.BoolValue destination_class_id = 39883017;
  ywrapper.BoolValue drop = 334671545;
  ywrapper.BoolValue drop_cancel = 232920976;
  ywrapper.BoolValue source_class_id = 13743256;
  ywrapper.BoolValue vrf = 33275360;
}
